src/app/pages/DashboardPage/ParatimeSnaphosts.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC, useCallback, useState } from 'react'
+import Grid from '@mui/material/Grid'
+import Typography from '@mui/material/Typography'
+import { COLORS } from '../../../styles/theme/colors'
+import { DurationSelect } from './DurationSelect'
+import { TransactionsChartCard } from './TransactionsChartCard'
+import { ChartDuration } from '../../utils/chart-utils'
+import { useTranslation } from 'react-i18next'
+import { useConstant } from '../../hooks/useConstant'
+
+export const ParatimeSnaphosts: FC = () => {
+  const { t } = useTranslation()
+  const defaultChartDurationValue = useConstant<ChartDuration>(() => ChartDuration.TODAY)
```

Comment:
how that hook works? can't we just use ChartDuration.TODAY ?

Reply:
It works the same way if you would define a variable outside of the component. But since we are keeping types and styles in the same file it's messy. I would like to keep it this way.